src/compiler/crystal/compiler.cr

```diff
@@ -225,9 +226,7 @@ module Crystal
       bc_flags_changed
     end
 
-    private def codegen(program : Program, node, sources, output_filename)
-      @link_flags = "#{@link_flags} -rdynamic"
-
+    private def codegen(program, node : ASTNode, sources, output_filename)
```

Comment:
It was superfluous and made the line longer. `ASTNode` is required to distinguish the override, `Program` is just repeating the information contained in the name `program`.